### PR TITLE
Destroy synchronization when user selects interval 0

### PR DESCRIPTION
**Pull request by xavijam**
Fixes https://github.com/CartoDB/cartodb/issues/9927.

CR: @matallo 
cc @javitonino

### CODE REVIEW

**Comment by xavijam** [comment]
I've spotted one thing in the builder side, let me fix it before CR @nobuti.

**Comment by xavijam** [comment]
Moving CR to @matallo, please man 💋 .

**Comment by matallo** [comment]
roger!

**Comment by xavijam** [comment]
ping-pong!

File: lib/assets/test/spec/cartodb3/data/query-rows-collection.spec.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-// var _ = require('underscore');
+// var _ = require('underscore')
```
Comment (matallo) [inline lib/assets/test/spec/cartodb3/data/query-rows-collection.spec.js]: shall we keep commented lines?

Comment (xavijam) [inline lib/assets/test/spec/cartodb3/data/query-rows-collection.spec.js]: no, let me remove it.

**Pull request merged**